README.md

```diff
@@ -76,103 +88,156 @@ Of course, this Guide can’t replace professional advice. But talk to anyone wh
+
+Sounds great, right? But wait just a minute. Let’s get into some of those conditions and limitations on your option agreement we mentioned above:
+
+- 🔸 **Vesting**: One of the most significant conditions is that you usually “earn” rights to the shares or options over time or under certain events. This is called **vesting**.
     - Vesting usually occurs according to a **vesting schedule**. You vest only while you work for the company. For example, it is very common to have stock or options vest over a period of four years, a bit at a time, where none of it is vested at first, and all of it is vested after four years.
     - Vesting schedules can also have a **cliff**, where until you work for a given amount of time, you are 0% vested. For example, if your equity award had a one-year cliff and you only worked for the company for 11 months, you would not get anything, since you have not vested in any part of your award. Similarly, if the company is sold within a year, depending on what your paperwork says, you may also receive nothing on the sale of the company.
     - A very common vesting schedule is vesting over 4 years, with a 1 year cliff. This means you get 0% vesting for the first 12 months, 25% vesting at the 12th month, and 1/48th (2.08%) more vesting each month until the 48th month. For example, if you leave just before a year is up, you get nothing, but if you leave after 3 years, you get 75%.
-    - Vesting might also occur in certain situations. You may have [**acceleration**](http://www.danshapiro.com/blog/2012/04/vesting-is-a-hack/), where vesting is triggered if a company is sold (**single trigger**) or if it’s sold and you’re fired (**double trigger**). This is common for founders and not so common for employees.
-- Grants for **advisors** typically vest over a shorter period than employee grants, often two years. Advisor grants also typically have a longer exercise window post termination of service. Typical terms for advisors, including equity levels, are available from the 📥 Founder Institute’s [Founder/Advisor Standard Template (FAST)](https://fi.co/contents/fast#).
+    - Vesting *can* be triggered outside of these timelines. You may have [**acceleration**](http://www.danshapiro.com/blog/2012/04/vesting-is-a-hack/), where vesting is triggered if a company is sold (**single trigger**) or if it’s sold and you’re fired (**double trigger**). This is common for founders and not so common for employees.
+- **Options expire**: You need to know how long the **exercise window** (the time during which you can buy shares at the strike price) will be open.
+		- Options are only exercisable for a fixed period of time, typically seven to ten years as long as you are working for the company.
+		- ❗ Importantly, options can expire after you quit working for the company. Often, the expiration is **90 days** after termination of service, making the options effectively worthless if you cannot exercise before that point.
```

Comment:
Could use more color on how to plan ahead for this / why it's important to plan ahead. Also add encouragement for employees to find this out on day 1.

Reply:
✔️